Makefile

```diff
@@ -25,7 +25,7 @@ DID_ELEMENT_SIDETREE_REQUEST_URL ?= https://element-did.com/api/v1/sidetree/requ
 SANDBOX_CLI_IMAGE_NAME       ?= trustbloc/sandbox-cli
 
 # TrustBloc core k8s deployment scripts https://github.com/trustbloc/k8s
-TRUSTBLOC_CORE_K8S_COMMIT=5652a580089b9b32f14ca3b31ab1f9982214d347
+TRUSTBLOC_CORE_K8S_COMMIT=38a569e8d344a22ad2873933282e4eaba96b9a11
```

Comment:
@alialkhalidi Do we need this to updated to [2a0c93d06e8d65a8bb73f776e1914df545d9023f](https://github.com/trustbloc/k8s/commit/2a0c93d06e8d65a8bb73f776e1914df545d9023f) ?

Reply:
yup, updated again.